### ISSUE TITLE
Image editor not functional but no error

### ISSUE THREAD

**Issue by Proofer** [BlueGreenMagick/Image-Editor#4]
Just enabled it in Anki 2.1.37 today. I previously pasted an image into a field in card-edit mode. Then I enabled the add-on. Then in a later card-edit when I right-click on the image and choose Edit Image, I get a new Image Annotation panel that shows only black or dark gray, with menu titles that are not responsive (the titles highlight but no commands drop down), with only the buttons at the bottom responsive -- so I can exit by clicking the Discard button.

Would love to have this working -- will provide any help that I can.

**Comment by BlueGreenMagick** [comment]
Sorry for the late reply. 

Could you try installing [AnkiWebview Installer](https://ankiweb.net/shared/info/31746032) and left click the annotate window, then go to console and see if there's any errors?

**Comment by Proofer** [comment]
The link is to AnkiWebview *Inspector*. What/where is the annotate window? What/where is the console?

**Comment by BlueGreenMagick** [comment]
Sorry about the typo! AnkiWebview Inspector is correct. Can you click `Edit Image` context menu on any image, then on the image editing pop-up window, right click somewhere inside and you should see `inspect` menu. 

When you click on the inspect menu, you should see the dev tool on the main Anki window. On the dev tools, there is a `console` tab on the top, next to `inspect`. If there are any errors, could you share them here?

**Comment by Proofer** [comment]
I opened Anki today and there was an Image Editor update, which I approved, then quit and re-opened Anki.

Things have changed. Now after I  right-click an image and choose Edit Image, I get an Annotate Image tab that shows the image. But right-clicking doesn't provide an inspect menu, nor do I see one anywhere. Right-clicking in the image area brings up a menu with clipboard functions, delete, group, ungroup, and layering functions.

However, my original problem appears to have been solved!
